docs/content/reference/video.md

```diff
@@ -24,25 +24,30 @@ This gives the best compression ratio, reducing file sizes and bandwidth require
 snippet: archetypes/video_auto_frames
 
 ## `AssetVideo` limitations
-Video support is new in Rerun, and has several limitations:
+Video support is new in Rerun, and has a few limitations:
 
-* [#7298](https://github.com/rerun-io/rerun/issues/7298): Video playback only works in the web viewer
 * [#7354](https://github.com/rerun-io/rerun/issues/7354): Only the MP4 container format is supported
+* [#7298](https://github.com/rerun-io/rerun/issues/7298): Only the AV1 codec is supported in the native viewer
 * [#5181](https://github.com/rerun-io/rerun/issues/5181): There is no audio support
+* [#7594](https://github.com/rerun-io/rerun/issues/7594): HDR video is not supported
 * There is no video encoder in the Rerun SDK, so you need to create the video file yourself
-* Only a limited sets of codecs are supported (see below)
+* A limited sets of codecs are supported on web (see below)
 
-## Web viewer support
-As of writing, playback of `AssetVideo` is only supported on the web viewer.
-Native video playback is coming, and can be tracked [in this GitHub issue](https://github.com/rerun-io/rerun/issues/7298).
+## Streaming video
+Rerun does not yet support streaming video support. You can work around this limitation by logging many small `AssetVideo`s to the same Entity Path. See [#7484](https://github.com/rerun-io/rerun/issues/7484) for more.
+
+## Native support
+In the native viewer, AV1 is the only supported codec. H.264 is coming soon ([#7298](https://github.com/rerun-io/rerun/issues/7298)).
 
-Video playback is done using the browser's own video decoder, so the supported codecs depend on your browser.
+
+## Web viewer support
+Video playback in the Rerun Web Viewer is done using the browser's own video decoder, so the supported codecs depend on your browser.
 
 Overall, we recommend using Chrome or another Chromium-based browser, as it seems to have the best video support as of writing.
 
 When choosing a codec, we recommend [AV1](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Video_codecs#av1),
 as it seems to have the best overall playback support while also having very high compression quality.
-Since AV1 is patent-free, it is also likely the first codec we will support in the native viewer.
+AV1 is also patent-free, and is the only codec we currently support in the native viewer (see [#7298](https://github.com/rerun-io/rerun/issues/7298)). H.264/avc is another popular choice, and native support for that is coming soon.
```

Comment:
Suggestion: Let's add a `## Codec support` section with this block at the start and then subsection `### Native` & `Web viewer` under it
(is it `native` or `desktop`? I noticed some folks don't have `native` as a term ingrained we do coming from systems programming languages)

Reply:
I still think this section is native/web independent and should go directly under the top `## Codec support` 🤷

Reply:
:+1: